codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +138,114 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        // TODO: Is this a good size ?
+        private static final int INITIAL_SIZE = 64;
```

Comment:
Sure

> Am 15.06.2016 um 19:32 schrieb Scott Mitchell <notifications@github.com>:
> 
> In codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java:
> 
> > @@ -213,4 +138,114 @@ private static void insert(Node root, int symbol, int code, byte length) {
> >              current.children[i] = terminal;
> >          }
> >      }
> > +
> > +    private static final class DecoderProcessor implements ByteProcessor {
> > +        // TODO: Is this a good size ?
> > +        private static final int INITIAL_SIZE = 64;
> hard to say and will depend upon the use case ... can we make the initial size configurable via the constructors?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>